system/apps_ds213/58_gabmini/source/app_ds213.lds

```diff
@@ -2,7 +2,7 @@ MEMORY
 {
   null (rwx): ORIGIN = 0x00001000, LENGTH = 4K
   rom (rx) : ORIGIN = 0x08013100, LENGTH = 40K
-  ram (rwx) : ORIGIN = 0x20002800, LENGTH = 28K
+  ram (rwx) : ORIGIN = 0x20002900, LENGTH = 28K
```

Comment:
Thank you very much for these fixes! It seems that we need some protection mechanism in the flashing code to verify if the user app does not overlap the system code.